ksqldb-engine/src/main/java/io/confluent/ksql/engine/TransientQueryCleanupService.java

```diff
@@ -100,9 +110,18 @@ public void setQueryRegistry(final QueryRegistry queryRegistry) {
     this.queryRegistry = queryRegistry;
   }
 
-  @SuppressFBWarnings(value = "EI_EXPOSE_REP2")
   public void setLocalCommandsQueryAppIds(final Set<String> ids) {
-    this.localCommandsQueryAppIds = Optional.of(ids);
+    localCommandsQueryAppIds = ImmutableSet.copyOf(ids);
+    final ImmutableSet.Builder<String> builder = ImmutableSet.builder();
+    for (String id : ids) {
```

Comment:
Just curious about this extra space before `:` (after `id`). Is this accepted code style in this project? Learning not pointing :)

Reply:
Looks like so. I searched for `\(.*:.*\) \{` in codebase and resulted in lots of existing code.